packages/k6-tdk/src/api/graph/v1/me.ts

```diff
@@ -0,0 +1,19 @@
+import { RefinedResponse } from 'k6/http';
+
+import { Request } from '@/utils/http';
+
+
+export class Me {
+  protected request: Request;
+  constructor(request: Request) {
+    this.request = request;
+  }
+
+  drives(): RefinedResponse<'text'> {
+    return this.request('GET', '/graph/v1.0/me/drives');
+  }
```

Comment:
Graph provides 2 endpoints for drives: `/drives` and `/me/drives`. Since you're requesting `/me/drives` here, does it make sense to reflect that in the method name?

Reply:
Ahh, so we're already in the `me`-scope here. I was just confused because `me()` is in this class as well, so that would be `me.me()` then. But it actually makes sense 👍